Search/SearchQueryBuilder.php

```diff
@@ -75,6 +75,34 @@ public function indexes(array $indexes)
     }
 
     /**
+     * Set the sort Field.
+     *
+     * @param string $sort
+     *
+     * @return SearchQueryBuilder
+     */
+    public function sort($sort)
+    {
+        $this->searchQuery->setSort($sort);
+
+        return $this;
+    }
+
+    /**
+     * Set the order.
+     *
+     * @param string $order
+     *
+     * @return SearchQueryBuilder
+     */
+    public function order($order)
+    {
+        $this->searchQuery->setOrder($order);
+
+        return $this;
+    }
```

Comment:
Remove this method. You should be able to have many orderings in a query. See above.